example/podspec.yaml

```diff
@@ -5,6 +5,8 @@ ext:
 - extensions.blacklist.BlacklistExtension:
     blacklist@env.prod:
     - frisbee
+    warning@env.staging:
```

Comment:
I think the most common use case would be the following:

- On staging branches, show warnings
- On master branches, raise an error and block the build

Does it make more sense to have two separate lists, or should we just have a setting like:
raise_errors@env.prod: true

And when it's not `raise_errors: true` it will just output log messages. Happy to go either way, I just think we probably don't want to duplicate the lists.

Reply:
My thought was it was better to just have two explicit lists.

We can use yaml to make it easier to duplicate the lists:

```
  blacklist: &blacklist
  - frisbee
  warning: *blacklist
```

Reply:
Right ... then though to satisfy my workflow, we'd want:

```
  blacklist@env.prod: &blacklist
  - frisbee
  warning: *blacklist
```

And then this would make it... warn on staging and raise errors on prod? It sort of seems confusing to me.

Reply:
I think part of my hesitation is the mix of YAML anchors with our custom `@env` syntax. I know `@env` untagging happens after the aliases have been expanded (at least, I think that's true), but I think this could be confusing to read...

Reply:
I don't mind using the `raise_errors` style flag. I think my other thought was that if we weren't able to separate the list then it would cause issues later... but we can use the same env tagging for the blacklist so that isn't really an issue like I was thinking.

I think I will go with the default being to raise errors though. Don't want people to use the blacklist and then only have it show messages on accident. We could do it like this:

```
blacklist:
- frisbee
raise_errors: false
raise_errors@env.prod: true
```

Reply:
LGTM ... thank you. :) That's a good approach. I think the mixing of the `@env` tagging and the YAML aliases makes things a little confusing, and it's a pattern I've tried to avoid elsewhere since I suspect most folks don't know which happens first.